src/actions/index.js

```diff
@@ -10,9 +11,22 @@ export const DECREMENT = 'DECREMENT';
 // is left to the reducer(s).
 
 export const increment = () => {
-  // Fill in this function
+  return {
+    type: INCREMENT,
+    payload: "plus"
```

Comment:
Notice how you're not using/accessing the action payload at all in your reducer. So, there's no need to pass on a payload here. 
```
  return {
    type: INCREMENT
  }
```
is enough.